Segment toggle doesn't work in cmd/clink

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Running `oh-my-posh toggle <segment>`  doesn't work under the cmd shell (with clink v1.6.13). It works as expected under (at least) powershell, nu, and bash.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/121028/1ead1233-e706-4627-bfe1-70d6e4a7230f)
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/121028/cc5732aa-e208-47da-a1ae-8c651c7aae46)


### Theme

{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "version": 2,
  "final_space": true,
  "shell_integration": true,
  "blocks": [
    {
      "type":      "prompt",
      "alignment": "left",
      "segments": [
        {
          "type":             "root",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:root",
          "template":         " \uF0E7"
        },
        {
          "type":             "path",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:path",
          "properties": {
            "style":                 "agnoster_short",
            "folder_icon":           "\u2026",
            "folder_separator_icon": "/",
            "max_depth": 3,
            "mapped_locations_enabled": true
          },
          "template": " \uE5FF {{ .Path }}"
        },
        {
          "type":             "git",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:vcs",
          "template":         " {{ .UpstreamIcon }}{{ .HEAD }}{{ if .Staging.Changed }} \uF046 {{ .Staging.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Working.Changed }} \uF044 {{ .Working.String }}{{ end }}{{ if gt .StashCount 0 }} \uF692 {{ .StashCount }}{{ end }}",
          "properties": {
            "fetch_stash_count":   true,
            "fetch_status":        true,
            "fetch_upstream_icon": true,
            "fetch_bare_info":     true
          }
        },
        {
          "type":             "svn",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:vcs",
          "properties": {
            "fetch_status": true
          },
          "template": " \uE0A0 {{.Branch}} r{{.BaseRev}} {{.Working.String}}"
        },
        {
          "type":             "perl",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:runtime",
          "template":         " \uE769 {{ .Full }}"
        },
        {
          "type":             "python",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:runtime",
          "properties": {
            "display_mode":      "files",
            "fetch_virtual_env": false
          },
          "template": " \uE235 {{ .Full }} "
        },
        {
          "type":             "java",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:runtime",
          "template":         " \uE738 {{ .Full }}"
        },
        {
          "type":             "go",
          "style":            "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground":       "p:text",
          "background":       "p:runtime",
          "template":         " \uE724 {{ .Full }}"
        }
      ]
    },
    {
      "type":      "prompt",
      "alignment": "right",
      "segments":  [
        {
          "type":             "text",
          "style":            "diamond",
          "leading_diamond":  "\uE0B6",
          "foreground":       "p:text",
          "background":       "p:execution",
          "template":         "\uDB81\uDEAD "
        },
        {
          "type":             "executiontime",
          "style":            "plain",
          "foreground":       "p:text",
          "background":       "p:execution",
          "template":         "{{ .FormattedMs }} ",
          "properties": {
            "style":     "galvestonms",
            "threshold": 1
          }
        },
        {
          "type":             "status",
          "style":            "diamond",
          "trailing_diamond": "\uE0B4",
          "foreground":       "p:text",
          "background":       "p:success",
          "template":         "{{ if eq .Code 0 }} \uF42E{{ else }} \uF467 ({{ .Code }}){{ end }}",
          "background_templates": [
            "{{ if ne .Code 0 }}p:failure{{ end }}"
          ],
          "properties": {
            "always_enabled": true
          }
        },
        // emit an invisible character to ensure that the trailing diamond in
        // the preceding segment is visible
        {
          "type":     "text",
          "style":    "plain",
          "template": "\u2800"
        }
      ]
    },
    {
      "type":      "prompt",
      "alignment": "left",
      "newline":   true,
      "segments": [
        {
          "type":       "text",
          "style":      "plain",
          "foreground": "default",
          "template":   "\u276F"
        }
      ]
    }
  ],
  "transient_prompt": {
    "background": "transparent",
    "foreground": "p:path",
    "template":   "\u276F"
  },
  "palettes": {
    "template": "tokyonight-storm",
    "list": {
      "campbell": {
        "text":       "#0C0C0C",
        "success":    "#13A10E",
        "failure":    "#C50F1F",
        "root":       "#E74856",
        "path":       "#F9F1A5",
        "vcs":        "#16C60C",
        "runtime":    "#61D6D6",
        "execution":  "#767676"
      },
      "chalk": {
        "text":       "#2B2D2E",
        "success":    "#80C470",
        "failure":    "#F24840",
        "root":       "#B23A52",
        "path":       "#B9AC4A",
        "vcs":        "#789B6A",
        "runtime":    "#44A799",
        "execution":  "#888888"
      },
      "one half dark": {
         "text":       "#282C34",
         "success":    "#98C379",
         "failure":    "#E06C75",
         "root":       "#E74856",
         "path":       "#E5C07B",
         "vcs":        "#98C379",
         "runtime":    "#56B6C2",
         "execution":  "#5A6374"
       },
      "tokyonight-storm": {
         "text":       "#24283B",
         "success":    "#9ECE6A",
         "failure":    "#F7768E",
         "root":       "#F7768E",
         "path":       "#E0AF68",
         "vcs":        "#9ECE6A",
         "runtime":    "#7AA2F7",
         "execution":  "#C0CAF5"
       },
      "solarized": {
        "base03":      "#002B36",
        "base02":      "#073642",
        "base01":      "#586E75",
        "base00":      "#657B83",
        "base0":       "#839496",
        "base1":       "#93A1A1",
        "base2":       "#EEE8D5",
        "base3":       "#FDF6E3",
        "yellow":      "#B58900",
        "orange":      "#CB4B16",
        "red":         "#DC322F",
        "magenta":     "#D33682",
        "violet":      "#6C71C4",
        "blue":        "#268BD2",
        "cyan":        "#2AA198",
        "green":       "#859900",
        "text":        "p:base02",
        "success":     "p:green",
        "failure":     "p:red",
        "root":        "p:orange",
        "path":        "p:yellow",
        "vcs":         "p:green",
        "runtime":     "p:blue",
        "execution":   "p:base1"
      }
    }
  }
}


### What OS are you seeing the problem on?

Windows

### Which shell are you using?

cmd

### Log output

```Shell
Version: 20.0.2

Shell: cmd (clink v1.6.13.eb61b2)

Prompt:

  ~/…/Local/clink-completions/completions                                                                     󰚭  ⠀
❯

Segments:

ConsoleTitle(false)                        -   0 ms
Root(false)                                -   0 ms
Path(true)                                 -   1 ms
Git(false)                                 -   0 ms
Svn(false)                                 -   2 ms
Perl(false)                                -   0 ms
Python(false)                              -   0 ms
Java(false)                                -   0 ms
Go(false)                                  -   0 ms
Text(true)                                 -   0 ms
Executiontime(false)                       -   0 ms
Status(true)                               -   0 ms
Text(true)                                 -   0 ms
Text(true)                                 -   0 ms

Run duration: 11.6752ms

Cache path: C:\Users\Mike\AppData\Local\oh-my-posh

Config path: C:\Users\Mike\AppData\Local\Programs\oh-my-posh\themes\mjcarman-powerline.omp.json

Logs:

[DEBUG] 10:13:08.527 shell.go:Getenv:326 → NO DATA
[TRACE] 10:13:08.528 shell.go:Getenv(TERM_PROGRAM) - 513.7µs
[DEBUG] 10:13:08.528 shell.go:Getenv:326 → C:\Users\Mike\AppData\Local
[TRACE] 10:13:08.528 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:13:08.528 shell_windows.go:CachePath() - 0s
[DEBUG] 10:13:08.529 shell.go:Getenv:326 → C:\Users\Mike\AppData\Local\Programs\oh-my-posh\themes\mjcarman-powerline.omp.json
[TRACE] 10:13:08.529 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 10:13:08.529 shell.go:Shell:601 → no shell name provided in flags, trying to detect it
[DEBUG] 10:13:08.533 shell.go:Shell:609 → process name: cmd.exe
[TRACE] 10:13:08.533 shell.go:Shell() - 3.9163ms
[TRACE] 10:13:08.533 shell.go:resolveConfigPath() - 3.9163ms
[TRACE] 10:13:08.533 shell.go:Init() - 5.961ms
[TRACE] 10:13:08.533 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 config.go:loadConfig() - 1.5578ms
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Shell() - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → NO DATA
[TRACE] 10:13:08.535 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[DEBUG] 10:13:08.535 text.go:Render:71 → Rendering template: tokyonight-storm
[TRACE] 10:13:08.535 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:WindowsRegistryKeyValue:217 → ColorizationColor(DWORD): 0xC41E4357
[TRACE] 10:13:08.535 shell.go:Shell() - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → clink v1.6.13.eb61b2
[TRACE] 10:13:08.535 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 10:13:08.535 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 10:13:08.535 text.go:Render:71 → Rendering template:
[TRACE] 10:13:08.535 shell.go:StatusCodes() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → 60
[TRACE] 10:13:08.535 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → NO DATA
[TRACE] 10:13:08.535 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Shell() - 0s
[TRACE] 10:13:08.535 shell.go:Shell() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:Flags() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → C:\Program Files\Eclipse Adoptium\jdk-17.0.11.9-hotspot\
[TRACE] 10:13:08.535 shell.go:Getenv(JAVA_HOME) - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → files
[DEBUG] 10:13:08.535 shell.go:Pwd:356 → C:\Users\Mike\AppData\Local\clink-completions\completions
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Perl
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Git
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Go
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → extensions: [.perl-version *.pl *.pm *.t]
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → folders: []
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → folders: []
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Root
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → home_enabled: false
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:Shell() - 0s
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Svn
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → files
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Java
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 shell.go:Getenv:326 → C:\Users\Mike\AppData\Local
[TRACE] 10:13:08.535 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → extensions: [pom.xml build.gradle.kts build.sbt .java-version .deps.edn project.clj build.boot *.java *.class *.gradle *.jar *.clj *.cljc]
[TRACE] 10:13:08.535 shell_windows.go:Root() - 0s
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → folders: []
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → home_enabled: false
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[DEBUG] 10:13:08.535 properties.go:GetString:28 → files
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 segment.go:SetEnabled:532 → Segment: Python
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 10:13:08.535 properties.go:GetString:28 → 
[DEBUG] 10:13:08.535 properties.go:GetString:28 → 
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → ~
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → fetch_virtual_env: false
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:GOOS() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → NO DATA
[DEBUG] 10:13:08.535 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → agnoster_short
[DEBUG] 10:13:08.535 properties.go:GetInt:40 → max_depth: 3
[DEBUG] 10:13:08.535 properties.go:GetString:28 → …
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → hide_root_location: false
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[DEBUG] 10:13:08.535 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 10:13:08.535 properties.go:GetString:28 → NO DATA
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → /
[DEBUG] 10:13:08.535 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 10:13:08.535 properties.go:GetString:28 → %s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → %s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → %s
[DEBUG] 10:13:08.535 properties.go:GetString:28 → %s
[TRACE] 10:13:08.535 shell.go:PathSeparator() - 0s
[DEBUG] 10:13:08.535 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.535 shell.go:HasFilesInDir(.perl-version) - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell_windows.go:IsWsl() - 0s
[TRACE] 10:13:08.535 shell.go:StackCount() - 0s
[DEBUG] 10:13:08.535 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.535 shell.go:HasFilesInDir(*.go) - 0s
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.535 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.535 shell.go:HasFilesInDir(*.pl) - 0s
[DEBUG] 10:13:08.535 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[TRACE] 10:13:08.535 shell.go:HasFilesInDir(*.py) - 0s
[DEBUG] 10:13:08.535 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.535 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.pm) - 519.3µs
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(go.mod) - 519.3µs
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(build.gradle.kts) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.ipynb) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.t) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(build.sbt) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(pyproject.toml) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(.java-version) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(.deps.edn) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(project.clj) - 0s
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(venv.bak) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(build.boot) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[DEBUG] 10:13:08.536 shell.go:HasFolder:438 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.java) - 0s
[TRACE] 10:13:08.536 shell.go:HasFolder(.venv) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.class) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFolder:438 → false
[TRACE] 10:13:08.536 shell.go:HasFolder(venv) - 0s
[DEBUG] 10:13:08.536 shell.go:HasFolder:438 → false
[TRACE] 10:13:08.536 shell.go:HasFolder(virtualenv) - 0s
[DEBUG] 10:13:08.536 shell.go:HasFolder:438 → false
[TRACE] 10:13:08.536 shell.go:HasFolder(venv-win) - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.gradle) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFolder:438 → false
[TRACE] 10:13:08.536 shell.go:HasFolder(pyenv-win) - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.jar) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.clj) - 0s
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell.go:HasFilesInDir:409 → false
[TRACE] 10:13:08.536 shell.go:HasFilesInDir(*.cljc) - 0s
[DEBUG] 10:13:08.536 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 10:13:08.536 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 10:13:08.536 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 10:13:08.536 shell_windows.go:DirIsWritable() - 1.0353ms
[TRACE] 10:13:08.536 shell_windows.go:Root() - 0s
[TRACE] 10:13:08.536 shell.go:Shell() - 0s
[TRACE] 10:13:08.536 shell.go:StatusCodes() - 0s
[TRACE] 10:13:08.536 shell_windows.go:IsWsl() - 0s
[DEBUG] 10:13:08.536 shell.go:TemplateCache:782 → environment: [=C:=C:\Users\Mike\AppData\Local\clink-completions\completions =clink.bin=C:\Program Files (x86)\clink =clink.id=14180 =clink.profile=C:\Users\Mike\AppData\Local\clink =clink.scripts= =ExitCode=00000000 ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\Mike\AppData\Roaming chocolateyinstall=C:\Users\Mike\AppData\Local\Programs\WingetUI\choco-cli CLINK_COMPLETIONS_DIR=C:\Users\Mike\AppData\Local\clink-completions\completions CLINK_DIR=C:\Program Files (x86)\clink clink_dummy_capture_env=  COLUMNS=120 CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=ROSHAR ComSpec=C:\WINDOWS\system32\cmd.exe DriverData=C:\Windows\System32\Drivers\DriverData GOPATH=C:\Users\Mike\go HOMEDRIVE=C: HOMEPATH=\Users\Mike IGCCSVC_DB=AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAyhtHuuKedUOJm4grDUpB2QQAAAACAAAAAAAQZgAAAAEAACAAAADZ2HB31adEfygus3MSV5VMRrsgn98VeoRfQYkiEWHSogAAAAAOgAAAAAIAACAAAAB4JwuQ8z71GuIVMKKbwKo5nBmg+tLgqztLQuysp+LM/mAAAAATd8VejTalG+dxvdPrAEVmudk/h/XIUiI7to7UT+rcItvPmZK4gLRQtRL3wJWhuXiwxtN1fDbqhF5ewePnLJ3pA2213Sb09TD7IlzoV9LgM28SrcHmoFI5lIdB/YMo7UtAAAAAtGbTyJRATuiOmJewGgWFzUG6O60snw1X/3zVZr9W9yi5RZc7xqzP7o1r5Sgp2MRdl90wAHCR5+ij20LRKulGLA== JAVA_HOME=C:\Program Files\Eclipse Adoptium\jdk-17.0.11.9-hotspot\ LINES=60 LOCALAPPDATA=C:\Users\Mike\AppData\Local LOGONSERVER=\\ROSHAR LS_COLORS=no=00:fi=00:di=01;32:ln=01;34:pi=40;33:so=01;35:do=01;35:bd=40;33:cd=40;33:or=40;31;01:ex=01;33:*.cmd=01;33:*.bat=01;33:*.exe=01;33:*.com=01;33:*.tar=01;36:*.tgz=01;36:*.arj=01;36:*.taz=01;36:*.lzh=01;36:*.zip=01;36:*.z=01;36:*.Z=01;36:*.gz=01;36:*.bz2=01;36:*.deb=01;36:*.rpm=01;36:*.jpg=01;35:*.png=01;35:*.gif=01;35:*.bmp=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.cdr=01;35:*.mpg=01;35:*.wmv=01;35:*.avi=01;35:*.fli=01;35:*.gl=01;35:*.dl=01;35: NUMBER_OF_PROCESSORS=12 OneDrive=C:\Users\Mike\OneDrive OS=Windows_NT PAGER=less -RF Path=C:\Program Files\Eclipse Adoptium\jdk-17.0.11.9-hotspot\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\7-Zip;C:\Program Files\IDM Computer Solutions\UltraEdit;C:\Program Files\IDM Computer Solutions\UltraCompare;C:\Program Files\dotnet\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\TortoiseSVN\bin;C:\Program Files\Go\bin;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\Program Files\Git\cmd;C:\Program Files\Calibre2\;C:\Users\Mike\AppData\Local\Programs\WingetUI\choco-cli\bin;C:\Users\Mike\AppData\Local\Microsoft\WindowsApps;C:\Users\Mike\AppData\Local\Microsoft\WinGet\Links;C:\Tools;C:\Tools\gnuwin32\bin;C:\Users\Mike\AppData\Local\Programs\oh-my-posh\bin;C:\Users\Mike\AppData\Roaming\Amazon;C:\Users\Mike\go\bin;C:\Users\Mike\AppData\Local\Microsoft\WinGet\Packages\junegunn.fzf_Microsoft.Winget.Source_8wekyb3d8bbwe;C:\Users\Mike\AppData\Local\Microsoft\WinGet\Packages\BurntSushi.ripgrep.MSVC_Microsoft.Winget.Source_8wekyb3d8bbwe\ripgrep-14.1.0-x86_64-pc-windows-msvc;C:\Users\Mike\AppData\Local\Microsoft\WinGet\Packages\sharkdp.fd_Microsoft.Winget.Source_8wekyb3d8bbwe\fd-v10.1.0-x86_64-pc-windows-msvc;C:\Users\Mike\go\bin PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC PERLDOC=-M Pod::Text::Color POSH_CURSOR_LINE=60 POSH_INSTALLER=winget POSH_SHELL_VERSION=clink v1.6.13.eb61b2 POSH_THEME=C:\Users\Mike\AppData\Local\Programs\oh-my-posh\themes\mjcarman-powerline.omp.json POSH_THEMES_PATH=C:\Users\Mike\AppData\Local\Programs\oh-my-posh\themes PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 151 Stepping 5, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=9705 ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PROMPT=$P$G PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\Mike\AppData\Local\Temp TMP=C:\Users\Mike\AppData\Local\Temp USERDOMAIN=ROSHAR USERDOMAIN_ROAMINGPROFILE=ROSHAR USERNAME=Mike USERPROFILE=C:\Users\Mike windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={0caa0dad-35be-5f56-a8ff-afceeeaa6101} WT_SESSION=e16f6e6c-c65b-4396-ab06-0191e2274925 ZES_ENABLE_SYSMAN=1]
[TRACE] 10:13:08.536 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.536 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.536 shell.go:User:494 → Mike
[TRACE] 10:13:08.536 shell.go:User() - 0s
[DEBUG] 10:13:08.536 shell.go:Host:511 → Roshar
[TRACE] 10:13:08.536 shell.go:Host() - 0s
[TRACE] 10:13:08.536 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.536 shell.go:Getenv:326 → NO DATA
[TRACE] 10:13:08.536 shell.go:Getenv(SHLVL) - 0s
[TRACE] 10:13:08.536 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.538 shell.go:CommandPath:553 → C:\Program Files\TortoiseSVN\bin\svn.exe
[TRACE] 10:13:08.538 shell.go:CommandPath(svn.exe) - 2.5995ms
[TRACE] 10:13:08.538 shell.go:HasCommand(svn.exe) - 2.5995ms
[TRACE] 10:13:08.538 shell.go:Pwd() - 0s
[ERROR] 10:13:08.538 shell.go:HasParentFilePath:697 → CreateFile .svn: The system cannot find the file specified.
[TRACE] 10:13:08.538 shell.go:HasParentFilePath(.svn) - 0s
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template:   {{ .Path }}
[TRACE] 10:13:08.538 shell.go:TemplateCache() - 0s
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: powerline
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[TRACE] 10:13:08.538 shell.go:Flags() - 0s
[TRACE] 10:13:08.538 shell.go:Pwd() - 0s
[TRACE] 10:13:08.538 shell.go:Flags() - 0s
[DEBUG] 10:13:08.538 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.538 shell.go:GOOS() - 0s
[TRACE] 10:13:08.538 shell.go:Flags() - 0s
[TRACE] 10:13:08.538 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.538 segment.go:SetEnabled:532 → Segment: Text
[TRACE] 10:13:08.538 shell.go:Pwd() - 0s
[TRACE] 10:13:08.538 shell.go:Pwd() - 0s
[TRACE] 10:13:08.538 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.538 shell_windows.go:58 → C:\Users\Mike
[DEBUG] 10:13:08.538 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.538 shell.go:GOOS() - 0s
[TRACE] 10:13:08.538 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.538 segment.go:SetEnabled:532 → Segment: Executiontime
[DEBUG] 10:13:08.538 segment.go:SetEnabled:532 → Segment: Status
[DEBUG] 10:13:08.538 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.538 shell.go:GOOS() - 0s
[TRACE] 10:13:08.538 shell.go:StatusCodes() - 0s
[DEBUG] 10:13:08.538 segment.go:SetEnabled:532 → Segment: Text
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: {{ .Code }}
[DEBUG] 10:13:08.538 properties.go:GetBool:22 → always_enabled: false
[TRACE] 10:13:08.538 shell.go:TemplateCache() - 0s
[TRACE] 10:13:08.538 shell.go:ExecutionTime() - 0s
[DEBUG] 10:13:08.538 properties.go:GetFloat64:34 → threshold: 1.000000
[TRACE] 10:13:08.538 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.538 properties.go:GetBool:22 → always_enabled: true
[TRACE] 10:13:08.538 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: ⠀
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: 󰚭
[DEBUG] 10:13:08.538 text.go:Render:71 → Rendering template: {{ if eq .Code 0 }} {{ else }}  ({{ .Code }}){{ end }}
[TRACE] 10:13:08.538 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:TemplateCache() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: diamond
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: plain
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: {{ if ne .Code 0 }}p:failure{{ end }}
[TRACE] 10:13:08.539 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: diamond
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: plain
[DEBUG] 10:13:08.539 shell_windows.go:TerminalWidth:111 → terminal width: 120
[TRACE] 10:13:08.539 shell_windows.go:TerminalWidth() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Flags() - 0s
[TRACE] 10:13:08.539 shell.go:Pwd() - 0s
[DEBUG] 10:13:08.539 shell_windows.go:58 → C:\Users\Mike
[TRACE] 10:13:08.539 shell.go:GOOS() - 0s
[DEBUG] 10:13:08.539 segment.go:SetEnabled:532 → Segment: Text
[TRACE] 10:13:08.539 shell.go:TemplateCache() - 0s
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: ❯
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[DEBUG] 10:13:08.539 shell.go:Getenv:326 → NO DATA
[TRACE] 10:13:08.539 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 10:13:08.539 text.go:Render:71 → Rendering template: plain
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[TRACE] 10:13:08.539 shell.go:Shell() - 0s
[DEBUG] 10:13:08.539 shell.go:Getenv:326 → C:\Users\Mike\AppData\Local
[TRACE] 10:13:08.539 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 10:13:08.539 shell_windows.go:CachePath() - 0s
[TRACE] 10:13:08.539 shell.go:Flags() - 0s
```

--- a/src/shell/scripts/omp.lua
+++ b/src/shell/scripts/omp.lua
@@ -6,6 +6,18 @@ if '::UPGRADE::' == 'true' then
     print(notice)
 end
 
+-- Cache PID
+
+function generateCustomUUID()
+    local template = 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+    return string.gsub(template, '[x]', function(c)
+        local v = math.random(0, 0xf)
+        return string.format('%x', v)
+    end)
+end
+
+os.setenv("POSH_PID", generateCustomUUID())
+
 -- Helper functions
 
 local function get_priority_number(name, default)
@@ -25,11 +37,11 @@ local function get_priority_number(name, default)
         return default
 	end
 end
-
+os.setenv("POSH_CURSOR_LINE", console.getnumlines())
 -- Environment variables
 
 local function environment_onbeginedit()
-    os.setenv("POSH_CURSOR_LINE", console.getnumlines())
+
 end
 
 -- Local state

